src/Symfony/Bundle/WebProfilerBundle/EventListener/WebDebugToolbarListener.php

```diff
@@ -60,6 +60,15 @@ public function isEnabled(): bool
         return self::DISABLED !== $this->mode;
     }
 
+    public function setMode(int $mode)
+    {
+        if (self::DISABLED !== $mode && self::ENABLED !== $mode) {
```

Comment:
```php
if(!\in_array($mode, [self::DISABLED, self::ENABLED])) {
throw new \InvalidArgumentException(sprintf('Invalid value "%s" provided for mode, use one of "%s" or "%s"', $mode, self::DISABLED, self::ENABLED));

Reply:
Isn't `in_array()` much slower?

Reply:
Yes. I woun't use `in_array()` here. The comparisons are faster and imho also more readable.

Reply:
Not agree for readability argument and even it's slower it's only in dev/debug environment.
My 2 cents ❤️